### ISSUE TITLE
Ability to add a map layer over the base map

### ISSUE THREAD

**Issue by pierzen** [hotosm/osm-tasking-manager#154]
A map layer would let show POI's and polygons over the map. For example, we could represent an Earthquake Epicenter  and a circle representing the most affected zone. Aside the map, a list could describe these layer objects.

The ability to provide file names for various languages would then provide translation.

**Comment by wonderchook** [comment]
File names for various languages you mean for the map layers specifically? 

I wonder if being able to rank individual squares by importance might be another way to accomplish this. Right now you just get any old square to work on versus the most high need ones.

**Comment by pgiraud** [comment]
"Priority areas" were one of the wanted features you put in the document
you first sent to François and I.

On Mon, Oct 14, 2013 at 10:47 AM, Kate Chapman notifications@github.comwrote:

> File names for various languages you mean for the map layers specifically?
> 
> I wonder if being able to rank individual squares by importance might be
> another way to accomplish this. Right now you just get any old square to
> work on versus the most high need ones.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/hotosm/osm-tasking-manager/issues/154#issuecomment-26242760
> .

## 

Pierre GIRAUD
Géomaticien, Analyste

Camptocamp France SAS
Savoie Technolac, BP 352
73377 Le Bourget du Lac, Cedex

Tel : 00 33 4 79 44 44 93
Mail : pierre.giraud@camptocamp.com
http://www.camptocamp.com

**Comment by pierzen** [comment]
This proposition is to let the editors provide a Data layer (osm, geojson, etc.) to be added over the map for a specific Task Manager job. This option would probably be used by the more experienced editors.  This could probably be integrated in a Priority functionality that lets editors provide informations in various ways  :
- Upload a data layer for the task
- From a map editing function similar to what is provided when we create a task, add the possibility to trace various polygons, add points, and give them some descriptions.

A data layer could be used in various ways to orient mapping activities, instruct for priority areas or specific instructions for an area. For example, we could show a layer of landuse areas and ask the mapper to find roads connecting to these places.  We could add polygons for priorities or particular instructions in a specific area of the task.

**Comment by wonderchook** [comment]
@pierzen okay that wasn't clear from the initial proposal. The idea would be so show the items in the editors? 

What would the tracing of polygons and points and adding descriptions do? Are you talking about a functionality like Ushahidi?

**Comment by pierzen** [comment]
Yes, like in Ushahidi or UMap, I would like a the ability to show an informative layer over the map. The addition of polygons, lines, points and descriptions is to give flexibility in adding specific  instructions for various areas.
